examples/next/faust-nx/wp-templates/index.js

```diff
@@ -0,0 +1,11 @@
+import page from './page';
+import archive from './archive';
+import single from './single';
+import home from './home';
+
+export default {
+  index: home,
```

Comment:
The home template should resolve to either `home` or `front-page`. This work hasn't been implemented yet but is documented here: https://wpengine.atlassian.net/browse/MERL-358

For the meantime, we may want to wait on implementing the home page until the above work has been completed or make this a Next.js file based page.

Reply:
I'll move it to a next file based page for now